src/main/resources/application.yml

```diff
@@ -42,11 +42,12 @@ consentmanager:
       url: ${CONSENT_MANAGER_URL}
   linkservice:
     url: ${LINK_SERVICE_URL}
-  cacheMethod: guava
+  cacheMethod: redis
```

Comment:
By default it can guava and in the AWS we can the environment variable to use Redis no?

Reply:
Sure.

Reply:
Made the changes